ntoskrnl/ex/work.c

```diff
@@ -576,13 +584,17 @@ ExpInitializeWorkerThreads(VOID)
     ExpCreateWorkerThread(HyperCriticalWorkQueue, FALSE);
 
     /* Create the balance set manager thread */
-    PsCreateSystemThread(&ThreadHandle,
-                         THREAD_ALL_ACCESS,
-                         NULL,
-                         0,
-                         NULL,
-                         ExpWorkerThreadBalanceManager,
-                         NULL);
+    Status = PsCreateSystemThread(&ThreadHandle,
+                                  THREAD_ALL_ACCESS,
+                                  NULL,
+                                  0,
+                                  NULL,
+                                  ExpWorkerThreadBalanceManager,
+                                  NULL);
+    if (!NT_SUCCESS(Status))
+    {
+        KeBugCheckEx(PHASE1_INITIALIZATION_FAILED, Status, 0, 0, 0);
+    }
 
     /* Get a pointer to it for the shutdown process */
     ObReferenceObjectByHandle(ThreadHandle,
```

Comment:
But here there's no complaint about the absence of return value check? 🤔🤔

Reply:
See the other comment